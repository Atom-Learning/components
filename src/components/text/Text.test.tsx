import { render, screen } from '@testing-library/react'
import { axe } from 'jest-axe'
import * as React from 'react'

import { Text } from '.'

describe(`Text component`, () => {
  it('renders some text', async () => {
    const { container } = render(<Text>TEXT</Text>)

    await screen.getByPlaceholderText('TEXT')

    expect(container).toMatchSnapshot()
    expect(await axe(container)).toHaveNoViolations()
  })
})
