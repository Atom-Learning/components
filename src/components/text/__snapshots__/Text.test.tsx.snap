// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Text component renders some text 1`] = `
.sxeds20 {
  color: var(--colors-tonal900);
  font-family: var(--fonts-sans);
  font-weight: 400;
  margin: 0;
  max-width: 60ch;
}

.sxeds20zdqo8--size-md {
  font-size: var(--fontSizes-md);
  line-height: 1.75;
}

.sxeds20zdqo8--size-md::before {
  content: '';
  margin-bottom: -0.5114em;
  display: table;
}

.sxeds20zdqo8--size-md::after {
  content: '';
  margin-top: -0.5114em;
  display: table;
}

<div>
  <p
    class="sxeds20 sxeds20zdqo8--size-md"
  >
    TEXT
  </p>
</div>
`;
